eng/Version.Details.xml

```diff
@@ -169,7 +169,11 @@
       <Uri>https://github.com/Microsoft/ApplicationInsights-dotnet</Uri>
       <Sha>53b80940842204f78708a538628288ff5d741a1d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.Current.Manifest-8.0.100.Transport" Version="8.0.0-rc.1.23411.2" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Current.Manifest-8.0.100" Version="8.0.0-rc.1.23411.2" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
There was accidentally a stable version build, this dependency doesn't work until there is a stable version because it will have a prerelease version as part of the package name.

Reply:
The stable build wasn't accidental. I've been working on a stable test build, trying to resolve all issues that we will hit when we start working on the GA build, this being one of them.
I don't think I follow, what part of the dependency name will change when we move to stable?

Reply:
Sorry, the issue is that the manifest packages other than the transport package have the prerelease version band encoded in the name of the package so if we change the dependency to Microsoft.NET.Workload.Emscripten.Current.Manifest-8.0.100 no darc packages will flow until the emsdk prerelease version string is removed and that currently only happens when stable versions are used.

Reply:
the transport manifest doesn't have the prerelease version in its name precisely so we can use it to get flow and it is especially tricky because at the moment we use the prerelease version string in the package version to reconstruct the actual package name via an ugly regex and all the parts need to agree on how to do that.